detector/wordpress.go

```diff
@@ -191,23 +211,65 @@ func extractToVulnInfos(pkgName string, cves []WpCveInfo) (vinfos []models.VulnI
 		}
 
 		var refs []models.Reference
+		if vulnerability.Poc != nil {
+			refs = append(refs, models.Reference{
+				Source: *vulnerability.Poc,
```

Comment:
Other values in Source filed is like CVE, UPSTREAM, trivy, etc. poc value in the sample is "<script>alert(1)</script>\r\n\r\n<script>alert(1)</script>\r\n\r\n<script>alert(1)</script>", it seems rather different kind of data.

How about include the poc value in the Optional map?

Reply:
I think Exploit is best instead of Reference. Or maybe it's Optional.

Reply:
This time, I implemented it using Optional.
The reason is that there are various patterns seen in the values entered in poc, such as 'only URL' or 'with detailed explanation', and it is difficult to match the Exploit struct attributes.

For a sample of poc, please see the following page.
https://wpscan.com/plugin/import-xml-feed/